lib/data_magic.rb

```diff
@@ -113,10 +121,30 @@ def self.search(terms, options = {})
     end
 
     # assemble a simpler json document to return
+    simple_result =
     {
       "metadata" => metadata,
       "results" => 	results
     }
+
+    if options[:command] == 'stats'
+      # Remove metrics that weren't requested.
+      aggregations = result['aggregations']
+      aggregations.each do |f_name, values|
+        # Count is not an interesting statistic
+        values.delete 'count'
```

Comment:
Doesn't `count` reflect the sample size after filters have been applied? I suspect it may be interesting/useful. Or have I misunderstood how this works?